Week2/js-exercises/ex5-myFavoriteHobbies.js

```diff
@@ -10,6 +10,12 @@
 
 function createHTMLList(arr) {
   // your code goes in here
+  const ul = document.getElementById('list');
+  const newArr = arr.map(hobby => {
```

Comment:
You don't need `newArr` for this to work
```suggestion
  arr.map(hobby => {
```

Reply:
forEach used new array removed